Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -536,8 +543,29 @@ class SearchViewController: SiteTableViewController, KeyboardHelperDelegate, Loa
         return super.tableView(tableView, heightForRowAt: indexPath)
     }
 
+    override func tableView(_ tableView: UITableView, willDisplayHeaderView view: UIView, forSection section: Int) {
+        guard let headerView = view as? ThemedTableSectionHeaderFooterView else { return }
+
+        headerView.applyTheme()
+    }
+
     override func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
-        return 0
+        guard section == SearchListSection.remoteTabs.rawValue,
+              hasFirefoxSuggestions else { return 0 }
+
+        return SiteTableViewControllerUX.HeaderHeight
```

Comment:
Will that work with dynamic type? Could we use automatic height instead?

Reply:
changed to use automatic height to work with dynamic type